erna/scripts/submit_runlist.py

```diff
@@ -0,0 +1,65 @@
+import click
+import pandas as pd
+
+from ..automatic_processing.database_utils import insert_new_jobs
+from ..utils import load_config
+from datetime import date
+
+from ..automatic_processing.database import (
+    database,
+    RawDataFile,
+    Jar,
+    XML,
+    Queue,
+    Job,
+)
+
+
+@click.command()
+@click.argument('runlist')
+@click.argument('jar')
+@click.argument('xml')
+@click.option(
+    '-p', '--priority', default=5, type=int,
+    help='Priority of the jobs, lower value means more important'
+)
+@click.option(
+    '-q', '--queue', default='fact_short',
+    help='Name of the queue to use'
+)
+@click.option('--config', '-c', help='Path to the yaml config file')
+def main(runlist, jar, xml, priority, queue, config):
+    '''
+    Submit automatic processing jobs for a given runlist
+
+    Arguments
+
+    RUNLIST: csv file with columns `night, run_id`
+    JAR: version of the fact-tools jar
+    XML: Name of the xml file to use
+
+    Jar and XML must be uploaded to the processing db using erna_upload
+    '''
+    config = load_config(config)
+
+    database.init(**config['processing_database'])
+
+    jar = Jar.select(Jar.id, Jar.version).where(Jar.version == jar).get()
+    xml = XML.get(name=xml, jar=jar)
+    queue = Queue.get(name=queue)
+
+    runs = pd.read_csv(runlist)
+    runs['year'] = runs['night'] // 10000
```

Comment:
Instead of doing this whole night conversation stuff, just to convert it back in the custom field anyways, why not add a isinstance in the NIghtField.db_value function to test if it gets an int and just take it. Would make this whole ordeal a little nicer. Other option is to use the the `fact.time.run2dt` function of pyfact in the list comprehension like this:

```
files = [
        RawDataFile.get(night=run2dt(str(row.night)), run_id=row.run_id)
        for row in runs.itertuples()
]
```
And remove all the calculation of the year, day stuff.

Reply:
I will not change the DB schema for this simple task